### PR TITLE
F343-update ui player and dont allow host and player labels to overlap

### PR DESCRIPTION
**Pull request by JohnnyPersichilli**

### CODE REVIEW

File: CranberryQueue/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1540,9 +1540,9 @@
         <image name="location.fill" catalog="system" width="64" height="56"/>
         <image name="magnifyingglass" catalog="system" width="64" height="56"/>
         <image name="peopleIcon" width="512" height="512"/>
+        <image name="plus" catalog="system" width="64" height="56"/>
```
Comment (JohnnyPersichilli) [inline CranberryQueue/Base.lproj/Main.storyboard]: this is a ui fix that should have been included in the last pr so i just put it here, changes the UIImage to a button to show opacity when clicked

File: CranberryQueue/Shared/PlayerView.swift [hunk 2]
```diff
@@ -15,21 +15,51 @@ protocol PlayerControllerDelegate: class {
 }
 
 class PlayerView: UIView, PlayerDelegate {
+    func updatePlayPauseUI(isPaused: Bool, isHost: Bool) {
+        if(isHost){
+            titleLabel.trailingAnchor.constraint(equalTo: playPauseImage.trailingAnchor, constant: 8).isActive = false
```
Comment (creiser2) [inline CranberryQueue/Shared/PlayerView.swift]: Why is this function above all the iboutlet declarations?

File: CranberryQueue/Shared/PlayerController.swift [hunk 3]
```diff
@@ -379,6 +371,12 @@ class PlayerController: NSObject, SPTAppRemotePlayerStateDelegate, RemoteDelegat
             else {
                 self.runTimer()
             }
+            
+            self.mapDelegate?.updateSongUI(withInfo: info)
+            self.queueDelegate?.updateSongUI(withInfo: info)
+            self.mapDelegate?.updatePlayPauseUI(isPaused: false, isHost: false)
+            self.queueDelegate?.updatePlayPauseUI(isPaused: false, isHost: false)
+            self.duration = info.duration
```
Comment (rolflocher) [inline CranberryQueue/Shared/PlayerController.swift]: Is self.duration being set too late? It looks like delegate.updateTimerUI expected to use the new value.

File: CranberryQueue/Shared/PlayerView.swift (hunk 2 above)
Comment (JohnnyPersichilli) [reply CranberryQueue/Shared/PlayerView.swift]: No clue, it has been there in every pr except ur last one so i kept it. Will change it tho if there is no purpose for it

**Pull request closed without merging**